package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bulbs-public-ads-manager",
-  "version": "8.2.0",
+  "version": "8.2.1",
```

Comment:
This has actually gotten far out of date.  The current version in master is actually `9.9.0`